charts/opentelemetry-operator/values.yaml

```diff
@@ -79,9 +79,11 @@ manager:
   resources:
     limits:
       cpu: 100m
+      ephemeral-storage: 300Mi
       memory: 128Mi
     requests:
       cpu: 100m
+      ephemeral-storage: 300Mi
```

Comment:
We shouldn't use `ephemeral-storage` by default.  We can include a value here, commented out, so that users know they are allowed to set `ephemeral-storage`, but let's not set a default value.